fabric-keybindings-v0/src/main/java/net/fabricmc/fabric/api/client/keybinding/FabricKeyBinding.java

```diff
@@ -37,9 +37,10 @@ protected FabricKeyBinding(Identifier id, InputUtil.Type type, int code, String
 	 * @return configured KeyCode
 	 */
 	public InputUtil.KeyCode getBoundKey() {
-		return ((KeyCodeAccessor) this).getKeyCode();
+		return ((KeyBindingAccess) this).getKeyCode();
 	}
 
+	@Deprecated // No reason to use
```

Comment:
Put this in the javadoc, `@reason`?

Reply:
will change this. there's another pr from kitten or sollace that actually fixed this.